tests/helpers.py

```diff
@@ -527,3 +543,37 @@ def run_io_with_rados_bench(**kw):
     logger.info(ret)
     logger.info("Finished radosbench")
     return ret
+
+
+def get_all_pvs():
+    """
+    Gets all pv in given namespace
+
+    Returns:
+         dict: Dict of all pvc in namespaces
+    """
+    ocp_pv_obj = ocp.OCP(
+        kind=constants.PV, namespace=defaults.ROOK_CLUSTER_NAMESPACE)
+    return ocp_pv_obj.get()
+
+
+@retry(AssertionError, tries=10, delay=5, backoff=1)
+def validate_pv_delete(pv):
+    """
+    validates if pv is deleted after pvc deletion
+
+    Args:
+        pv (str): pv from pvc to validates
+    Returns:
+        bool: True if deletion is successful
+    """
+    ocp_pv_list = get_all_pvs()
+    pv_all_list = ocp_pv_list['items']
+    if pv_all_list:
+        for item in pv_all_list:
+            if item['metadata']['name'] == pv:
+                raise AssertionError
```

Comment:
the docstring says this function return bool

Reply:
Updated !

Reply:
Why not returning False?